example/AppDemo.tsx

```diff
@@ -9,7 +9,9 @@ if (__DEV__) {
 const App: FC = () => {
   return (
     <RootProviders authConfig={authConfig}>
-      <RootStack />
+      <NoInternetToastProvider>
+        <RootStack />
+      </NoInternetToastProvider>
```

Comment:
Did you put this here (and not inside `RootProviders`) to give SDK users more of a choice about how to display no-internet message?  I'd like to keep this simpler in this first pass we're in, and put everything that is in the app shell inside RootProviders or RootStack.

Reply:
Yeah, I chose not to include it by default to allow the app developer the choice but I will update it.

Reply:
Updated